core/math/a_star.cpp

```diff
@@ -129,6 +129,17 @@ bool AStar::has_point(int p_id) const {
 	return points.has(p_id);
 }
 
+Array AStar::get_points()
+{
```

Comment:
Brace should be in the same line as method name.
`Array AStar::get_points() {`

Reply:
for reference http://docs.godotengine.org/en/stable/community/contributing/code_style_guidelines.html

Reply:
Thank you! my clang format wasnt set up properly it seems. Its fixed now ^^